earthaccess/auth.py

```diff
@@ -231,29 +234,29 @@ def _netrc(self) -> bool:
         try:
             my_netrc = Netrc()
         except FileNotFoundError as err:
-            print(f"No .netrc found in {os.path.expanduser('~')}")
-            raise err
+            raise FileNotFoundError(
+                f"No .netrc found in {os.path.expanduser('~')}"
+            ) from err
         except NetrcParseError as err:
-            print("Unable to parse .netrc")
-            raise err
+            raise NetrcParseError("Unable to parse .netrc") from err
```

Comment:
Are we re-raising because the messages from the library aren't that great? If we don't need to do any handling, can we remove the try/except?

Or was the idea to have log messages for cases when tracebacks are disabled? In which case, can we `logger.error` these messages?

Reply:
These exceptions would be swallowed [here](https://github.com/nsidc/earthaccess/pull/300/files#diff-28208126471c4332a472c11d7d8d595053abc2031e4b38b660753a28f79ba7acR56-R57) so I think  we should `logger.error` these before re-raising. What do you all think?

Reply:
I don't fully recall, but yeah, I think this was to make the message that was actually raised to the user more meaningful

Reply:
>These exceptions would be swallowed [here](https://github.com/nsidc/earthaccess/pull/300/files#diff-28208126471c4332a472c11d7d8d595053abc2031e4b38b660753a28f79ba7acR56-R57) so I think we should logger.error these before re-raising. What do you all think?

The link "here" is a perma-link, but it doesn't look like it points to what I thought it would. I thought these exceptions were going to be swallowed at the time I wrote the comment, likely I meant to link to [here](https://github.com/nsidc/earthaccess/pull/300/files#diff-28208126471c4332a472c11d7d8d595053abc2031e4b38b660753a28f79ba7acR65-R68), which didn't have debug logging at the time. I think this is resolved, please un-resolve if you catch any misunderstandings :)